core/camel-support/src/main/java/org/apache/camel/component/rabbitmq/SimpleDataHolderBean.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.component.rabbitmq;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class SimpleDataHolderBean {
```

Comment:
@rgannu we mentioned that we want this class to be in the component's package which means the component's folder (`components/camel-rabbitmq/src/main/java/org/apache/camel/component/rabbitmq`)

Reply:
oops. I wanted to also move into the camel-rabbitmq module only. The refactoring in intellij has moved it to somewhere and I didn't notice. Fixed it properly now. 
Also, as @oscerd was mentioning about the rebase problem I created again a new branch and moved this work to that. 
https://github.com/apache/camel/pull/4371

Is it possible to move the comment from this PR to 4371 PR? I do not know.